import os

LOG_FILE = 'src/database/log.txt'

def log_exception(err_source,error):
    pass



